sections/futures/MarketDetails/useGetMarketData.ts

```diff
@@ -45,14 +45,15 @@ const useGetMarketData = (mobile?: boolean) => {
 
 	const pastPrice = pastRates.find((price) => price.synth === marketAsset);
 
-	const fundingTitle = React.useMemo(
-		() => `${!fundingRate?.fundingRate && !!marketInfo ? 'Inst.' : '1H'} Funding Rate`,
-		[fundingRate, marketInfo]
-	);
+	const fundingTitle = React.useMemo(() => `${fundingRate?.fundingTitle ?? '1H Funding Rate'}`, [
```

Comment:
Should probably add this to translations

Reply:
agreed, I updated this